src/bloxone_client/api_client.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         self.configuration = configuration
 
         self.rest_client = RESTClientObject(configuration)
-        self.default_headers = {}
+        self.default_headers = configuration.default_headers
```

Comment:
the default_headers can be part of `api_client.py` itself. It doesn't have to be in configuration. The headers that we want to send by default (`x-infoblox-client`,`x-infoblox-sdk`)  should be on top of whatever existing default headers the user set. I think therefore the better location to do this would be just before call_api where you can append these headers to existing headers.

Reply:
Added them to param_serialize , function called before call_api
(x-infoblox-client,x-infoblox-sdk) appear above all headers defined by us . Headers like Accept Encoding , which are given by requests , are still above these !

Reply:
by "on top of" I meant in order of precendence.